README.md

```diff
@@ -394,3 +388,29 @@ If that returns `true`, you can launch the mesh scene editing session:
     artifactManager.accessMeshScene()
 ```
 
+## Known Issues
+
+### iOS 12 Storyboard Crash
+
+There is a certain configuration that can cause iOS 12 to crash, if ALL of the following conditions are met:
+* Your app and StreemKit share a cocoapod dependency
+* You have a `ViewController` with a non-frozen  `struct` or `enum` as a stored property, from either `StreemKit` or a shared dependency
+* You instantiate the `ViewController` from a Storyboard
```

Comment:
I am confused about this one?

Reply:
Its a pretty tough issue... i did the best i could to explain, but please let me know which parts are confusing, and happy to reword.  The bug occurs under the three conditions mentioned on iOS 12 and below.

Reply:
Only the storyboard part I was confused about. 
It was not clear to me why this would cause the error. 
But maybe it is not the storyboard that causes the issue but the three combined?

Reply:
Exactly!  Above we mention that ALL the following conditions need to be met... yes, crazy bug.